DB/projects.json

```
@@ -4779,5 +4779,16 @@
         ]
       }
     ]
-  }
+  },
+  {
+  "github_username": "Prakhar-Shankar",
+  "Projects": [
+    {
+      "github_url": "https://github.com/Prakhar-Shankar/crypto-tracker",
+      "title": "crypto-tracker",
+      "description": "A website which tracks and provides details about the real time trend in crypto currency.",
+      "tech": ["Reactjs", "javascript"]
```

convert to small case 
```suggestion
      "tech": ["reactjs", "javascript"]
```

---

I have made the necessary changes.